### PR TITLE
Add missing variables and change to be more intuitive to use

### PR DESCRIPTION
**Pull request by GC-20-20**
## Summary
Add missing variables and change to be more intuitive to use
## Impact
Variables that do cause compilation to fail
## Testing
local test

### CODE REVIEW

File: examples/serialrx/Kconfig [hunk 1]
```diff
@@ -39,22 +39,22 @@ config EXAMPLES_SERIALRX_DEVPATH
 
 choice
 	prompt "Output method"
-	default EXAMPLES_SERIALRX_PRINTHYPHEN
+	default EXAMPLES_SERIALRX_PRINTSTR
 
-config EXAMPLES_SERIALRX_PRINTHYPHEN
```
Comment (xiaoxiang781216) [inline examples/serialrx/Kconfig:44]: why change the order, let's keep as before

File: examples/serialrx/Kconfig [hunk 2]
```diff
@@ -39,22 +39,22 @@ config EXAMPLES_SERIALRX_DEVPATH
 
 choice
 	prompt "Output method"
-	default EXAMPLES_SERIALRX_PRINTHYPHEN
+	default EXAMPLES_SERIALRX_PRINTSTR
```
Comment (xiaoxiang781216) [inline examples/serialrx/Kconfig:42]: why change the default setting

File: examples/serialrx/serialrx_main.c [hunk 3]
```diff
@@ -118,8 +118,7 @@ int main(int argc, FAR char *argv[])
   while (cnt < bytecount)
     {
 #ifdef CONFIG_EXAMPLES_SERIALRX_BUFFERED
-      size_t n = fread(buf, 1, 26, f);
```
Comment (xiaoxiang781216) [inline examples/serialrx/serialrx_main.c:121]: why make the follow change

File: examples/serialrx/Kconfig (hunk 2 above)
Comment (GC-20-20) [reply examples/serialrx/Kconfig:42]: When I opened this application for the first time, it was very confusing for me to not have read the source code of the program and not have any display after using another serial port to send data, after all, there is no explanation in the HELP. So I would like to change the default value to EXAMPLES_SERIALRX_PRINTSTR so that later learners can be more intuitively clear that the previous operation is correct.

File: examples/serialrx/serialrx_main.c (hunk 3 above)
Comment (GC-20-20) [reply examples/serialrx/serialrx_main.c:121]: I'm very sorry, this change is really optional, this 26 I double-checked a few times without finding any clues, then I changed it to a simple number. Maybe it helps.

Comment (xiaoxiang781216) [reply examples/serialrx/serialrx_main.c:121]: Ok, but after you finish the investigation is better to remove the temp change. If you think the temp change is worth to upstream, please split to the different patch. It's important to ensure one patch fix one problem, so the reviewer could understand your change quickly.

Comment (GC-20-20) [reply examples/serialrx/serialrx_main.c:121]: Okay, I see what you mean.I'd still like to ask, but would these small changes be a waste of your time.After all, it doesn't change the logic of the code

Comment (xiaoxiang781216) [reply examples/serialrx/serialrx_main.c:121]: You can have one PR which contain more than one patch

**Review by jerpelea — CHANGES_REQUESTED** [review]
a commit message would be nice

File: examples/serialrx/serialrx_main.c [hunk 4]
```diff
@@ -173,7 +172,6 @@ int main(int argc, FAR char *argv[])
 
       UNUSED(eof);
     }
-
```
Comment (xiaoxiang781216) [inline examples/serialrx/serialrx_main.c:176]: revert this change

File: examples/serialrx/serialrx_main.c [hunk 5]
```diff
@@ -118,8 +118,7 @@ int main(int argc, FAR char *argv[])
   while (cnt < bytecount)
     {
 #ifdef CONFIG_EXAMPLES_SERIALRX_BUFFERED
-      size_t n = fread(buf, 1, 26, f);
-      cnt++;
+      size_t n = fread(buf, 1, 1, f);
```
Comment (xiaoxiang781216) [inline examples/serialrx/serialrx_main.c:121]: why not write 26 bytes.

**Comment by xiaoxiang781216** [comment]
please squash your patch and update the commit message to reflect your change.
